README.md

```diff
@@ -2,24 +2,24 @@
 vue.js pdf viewer
 
 ## Install
-```
+```bash
 npm install --save vue-pdf
 ```
 
 ## Example - basic
-```
+```js
 <template>
-  <pdf src="./static/relativity.pdf"></pdf>
+  <vue-pdf src="./static/relativity.pdf" />
```

Comment:
good, just for the record: [Self-closing components](https://vuejs.org/v2/style-guide/#Self-closing-components-strongly-recommended)